modules/ztp-sno-du-enabling-workload-partitioning.adoc

```diff
@@ -76,8 +76,8 @@ sh-4.4# pgrep systemd | while read i; do taskset -cp $i; done
 .Example output
 [source,terminal]
 ----
-pid 1's current affinity list: 0-3
-pid 938's current affinity list: 0-3
-pid 962's current affinity list: 0-3
-pid 1197's current affinity list: 0-3
+pid 1's current affinity list: 0-1,52-53
+pid 938's current affinity list: 0-1,52-53
+pid 962's current affinity list: 0-1,52-53
```

Comment:
Basically if the partitioning is configured correctly both apps (e.g ovn) and system components (e.g systemd) should stick to the same set of cpus 

e.g output from my system 
```
[root@cnfdf27 core]# pgrep ovn | while read i; do taskset -cp $i; done
pid 17667's current affinity list: 0,1,56,57
pid 18331's current affinity list: 0,1,56,57
pid 18519's current affinity list: 0,1,56,57
pid 18675's current affinity list: 0,1,56,57
pid 19208's current affinity list: 0,1,56,57
pid 19775's current affinity list: 0,1,56,57
[root@cnfdf27 core]# pgrep systemd | while read i; do taskset -cp $i; done
pid 1's current affinity list: 0,1,56,57
pid 2479's current affinity list: 0,1,56,57
pid 2505's current affinity list: 0,1,56,57
pid 2924's current affinity list: 0,1,56,57
pid 304651's current affinity list: 0,1,56,57
```


So in your case in your just replace `0-3` with `0-1,52-53`

Reply:
If "apps" is referring to customer application pods, those should be running on the isolated cores: 2-51,54-102. ovn is part of the OpenShift infrastructure.

Reply:
Yeah meant OpenShift infrastructure apps (ovn)

Reply:
@amolnar-rh not part of the your change but `user applications CPU pinning` should say `OpenShift infrastructure applications CPU pinning`

Reply:
Or maybe just combine them both? e.g 

"checking partition with taskset"
```
pgrep "systemd|ovn" | while read i; do echo "CPUSet $(taskset -cp $i | grep -Po '[0-9]+[-,]+[0-9]+.*') for process $(ps -p $i -o comm=)"; done
CPUSet 0,1,56,57 for process systemd
CPUSet 0,1,56,57 for process systemd-journal
CPUSet 0,1,56,57 for process systemd-udevd
CPUSet 0,1,56,57 for process systemd-logind
CPUSet 0,1,56,57 for process ovnkube-identit
CPUSet 0,1,56,57 for process ovnkube-identit
CPUSet 0,1,56,57 for process ovnkube
CPUSet 0,1,56,57 for process ovn-controller
CPUSet 0,1,56,57 for process ovn-northd
CPUSet 0,1,56,57 for process ovnkube
CPUSet 0,1,56,57 for process systemd
```

Reply:
@pixelsoccupied I didn't combine them, might be handier to have them separate, no?

However, I updated the cpusets and wording as discussed/suggested.

Reply:
Previously (until 4.13 i think?) we had to configure in two places to activated two types of pinning...so it sort of made sense to have it split. But either way it's fine!